nni/typehint.py

```diff
@@ -5,18 +5,11 @@
 Types for static checking.
 """
 
-__all__ = [
-    'Literal', 'TypedDict',
-    'Parameters', 'SearchSpace', 'TrialMetric', 'TrialRecord',
-]
-
-import sys
-from typing import Any, Dict, List, TYPE_CHECKING
-
-if TYPE_CHECKING or sys.version_info >= (3, 8):
```

Comment:
By the way, I think this doesn't work on Python 3.7.

Maybe it should be ``not TYPE_CHECKING``.